```
mingcongbai@JellyFocals [ ~ ] $ python3
Python 3.10.18 (main, Jun  4 2025, 08:17:53) [GCC 14.3.0 20250523 (AOSC OS, Core)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pytest_mock
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.10/site-packages/pytest_mock/__init__.py", line 1, in <module>
    from pytest_mock.plugin import AsyncMockType
  File "/usr/lib/python3.10/site-packages/pytest_mock/plugin.py", line 21, in <module>
    import pytest
ModuleNotFoundError: No module named 'pytest'
```

And why was `setuptools-scm` marked as a runtime dependency (which it should never have been)?

--- a/lang-python/pytest-mock/autobuild/defines
+++ b/lang-python/pytest-mock/autobuild/defines
@@ -1,0 +1,9 @@
+PKGNAME=pytest-mock
+PKGSEC=python
+PKGDEP="setuptools pytest"
+BUILDDEP="setuptools-scm"
+PKGDES="Thin-wrapper around the mock package for easier use with pytest"
+
+ABTYPE=pep517
+ABHOST=noarch
+NOPYTHON2=1

--- a/lang-python/pytest-mock/spec
+++ b/lang-python/pytest-mock/spec
@@ -1,0 +1,4 @@
+VER=3.15.1
+SRCS="git::commit=tags/v${VER};copy-repo=true::https://github.com/pytest-dev/pytest-mock"
+CHKSUMS="SKIP"
+CHKUPDATE="anitya::id=11587"

